extra/register-service.rb

```diff
@@ -23,11 +21,6 @@
   puts 'The acount can be local or a domain account. If it is a domain account then use the domain\account syntax'
   user  = ask("Run this service as this user? ") {|u| u.default = default_user}
   pass1 = ask("Enter the user's password ") {|p| p.echo = "x"}
-  pass2 = ask("Reenter the password "){|p| p.echo = "x"}
```

Comment:
Not sure it's worth it?  It's quite annoying needing to enter the password twice here as it's pretty easy to change service password if you get it wrong...